pages/seo.js

```diff
@@ -1,15 +1,120 @@
 import React from "react";
 import Head from "next/head";
 
-const Seo = ({ title, description, url, date, article }) => {
+const Seo = ({
+  title,
+  description,
+  authorName,
+  url,
+  image_url,
+  publishedAt,
+  publishedTime,
+  readingTime,
+  article,
+}) => {
   return (
     <Head>
-      <meta charset="UTF-8" />
-      <meta name="robots" content="noindex, nofollow" />
-      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
       {article ? (
-        <script type="application/ld+json">
-          {`
+        <>
+          <title>
+            {title} | by {authorName} | published at {publishedAt} | Canopas
+          </title>
+          <meta charset="UTF-8" />
+          <meta name="robots" content="noindex, nofollow" />
+          <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1.0"
+          />
+          <meta
+            data-rh="true"
+            property="og:site_name"
+            content="Canopas blogs"
+          />
+          <meta data-rh="true" property="og:type" content="article" />
+          <meta
+            data-rh="true"
+            property="article:published_time"
+            content={publishedTime}
+          />
+          <meta data-rh="true" property="title" content={title} />
+          <meta data-rh="true" property="og:title" content={title} />
+          <meta data-rh="true" property="twitter:title" content={title} />
+          <meta
+            data-rh="true"
+            name="description"
+            content={description.substring(0, 200) + "..."}
```

Comment:
I think it will not be effective. Let's add and use the `meta_description` field from Strapi